doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -1,422 +1,496 @@
+   |                           |   single:environment variable; CRM_alert_status                |
+   |                           |   single:CRM_alert_status                                      |
+   |                           |                                                                |
+   |                           | A numerical code used by Pacemaker to represent the operation  |
+   |                           | result (``resource`` alerts only)                              |
+   +---------------------------+----------------------------------------------------------------+
+   | CRM_alert_exec_time       | .. index::                                                     |
+   |                           |   single:environment variable; CRM_alert_exec_time             |
+   |                           |   single:CRM_alert_exec_time                                   |
+   |                           |                                                                |
+   |                           | The (wall-clock) time, in milliseconds, that it took to        |
+   |                           | execute the action. If the action timed out,                   |
+   |                           | ``CRM_alert_status`` will be 2, ``CRM_alert_desc`` will be     |
+   |                           | "Timed Out", and this value will be the action timeout. May    |
+   |                           | not be supported on all platforms. (``resource`` alerts only)  |
+   +---------------------------+----------------------------------------------------------------+
+   | CRM_alert_attribute_name  | .. index::                                                     |
+   |                           |   single:environment variable; CRM_alert_attribute_name        |
+   |                           |   single:CRM_alert_attribute_name                              |
+   |                           |                                                                |
+   |                           | The name of the node attribute that changed (``attribute``     |
+   |                           | alerts only)                                                   |
+   +---------------------------+----------------------------------------------------------------+
+   | CRM_alert_attribute_value | .. index::                                                     |
+   |                           |   single:environment variable; CRM_alert_attribute_value       |
+   |                           |   single:CRM_alert_attribute_value                             |
+   |                           |                                                                |
+   |                           | The new value of the node attribute that changed               |
+   |                           | (``attribute`` alerts only)                                    |
+   +---------------------------+----------------------------------------------------------------+
```

Comment:
It might be nice if a table of this size were arranged alphabetically by first column.

Reply:
It's organized by the type of alert it applies to (common variables at the top, then node alert variables, then fencing/resource alerts, etc.

But there was no secondary sort so I arranged them alphabetically within each type. Another column might make sense to make that more explicit, but I hate having to touch rST tables ...